### DISCUSSION TITLE
Artifact in Google Colaboratory

### DISCUSSION THREAD

**Discussion by Xixaus** [optuna/optuna#5131] [Q&A]
Hi, I would like to use human-in-loop optimization in Google Colaboratory. To run Dashboard I use this code: https://stackoverflow.com/questions/76033104/launching-optuna-dashboard-in-google-colaboratory.
Is there a way to insert a --artifact-dir ./artifact reference into the code?
Thank you for the answer

**Comment by nzw0301** [comment]
Hello, I suppose https://github.com/optuna/optuna-dashboard might be a more appropriate channel for optuna dashboard related questions.
